### PR TITLE
Enable CORS in Production

### PR DESCRIPTION
**Pull request by EndangeredMassa**
This change adds cors support from our specific frontend domain.

Corresponding frontend PR: https://github.com/ember-learn/ember-help-wanted/pull/83

### CODE REVIEW

File: src/server.js [hunk 1]
```diff
@@ -13,6 +15,9 @@ class Server {
   }
 
   start() {
+    app.use(cors({
+      origin: 'https://ember-help-wanted.netlify.com'
```
Comment (sivakumar-kailasam) [inline src/server.js]: May be pick this from an environment variable?

File: src/server.js [hunk 2]
```diff
@@ -1,4 +1,6 @@
 const express = require('express');
+const _ = require('lodash');
```
Comment (sivakumar-kailasam) [inline src/server.js]: Is this being used?

File: src/server.js (hunk 1 above)
Comment (EndangeredMassa) [reply src/server.js]: yeah, that's a better idea

**Comment by EndangeredMassa** [comment]
@sivakumar-kailasam updated!

**Pull request merged**